### DISCUSSION TITLE
profile picture not displaying on mobile devices

### DISCUSSION THREAD

**Discussion by cliu0013** [alshedivat/al-folio#543] [Q&A] [answered]
Thank you for this great template! I am trying to host my personal website using this template on github. I replaced assets/img/prof_pic.jpg with my own profile photo, and everything worked well on my PC's chrome, except that the picture is not showing up when I use chrome on my mobile device (ios). Hope somebody could help!

**Comment by kushasareen** [comment]
I'm having the same issue with Samsung Internet. The repo is at https://github.com/kushasareen/kushasareen.github.io. Were you ever able to resolve this?

**Comment by cliu0013** [comment]
The problem I encountered was due to my build environment. Basically I
tried to built the source code by myself without following the Readme step
by step. What I did was clone the project and built again from scratch as I
was only 10 mins into my project so it was quite efficient. I think if you
check out the make files in details you will find out the real problem, but
rebuild is a quick fix for me FYI.

On Thu, Jun 2, 2022 at 3:55 PM Kusha Sareen ***@***.***>
wrote:

> I'm having the same issue with Samsung Internet. The repo is at
> https://github.com/kushasareen/kushasareen.github.io. Were you ever able
> to resolve this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alshedivat/al-folio/discussions/543#discussioncomment-2873931>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK2GNDJY5IYGXN5JDZSKJHDVNEGSDANCNFSM5MZU4ZOQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Best,
Chengji

**Comment by kushasareen** [comment]
Thanks for your help! The build env pointed me in the right direction. For anyone in the future, the deployment docs at https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll suggests commenting out the "gem jekyll" and using "gem "github-pages" in the Gemfile but this caused a lot of issues for me, including causing pagination to break. Re-adding this fixes a lot of issues.

Thank again :)

**Reply by Zhendong-Wang** [reply]
@kushasareen Hi, I faced the same issue that my profile photo shows well on my Mac while it fails on my mobile phone and ipad. Would you like to tell me more details that how you solve the problem? 

Basically, I just fork, edit, commit and push. Thanks!

**Reply by rohandebsarkar** [reply]
@Zhendong-Wang took a quick look at your repo. Looks like you have disabled `jekyll-imagemagic`, but have kept responsive images turned on. You have to either turn both on or turn both off.

**Reply by rohandebsarkar** [reply]
Just pushed a commit 3672e8d2b1ae19db2bc246b2aecc0931356a753d, now responsive image will depend on one variable only.

**Reply by Zhendong-Wang** [reply, marked as answer]
Wow, that actually solves the problem easily! Great thanks! I made the mistake by disabling `jekyll-imagemagic`, since one youtuber said that is going to make problems. lol. I think the repo is wonderful!
